src/SecTester.Scan/Har/HarContent.cs

```diff
@@ -0,0 +1,5 @@
+namespace SecTester.Scan.Har;
+
+public class HarContent

```

Comment:
Use `record` instead
```suggestion
public record Har

```